packages/common/src/ui/layout/tables/hooks/useColumns/useColumns.tsx

```diff
@@ -57,7 +57,8 @@ const getDefaultAccessor =
   ): ColumnDataAccessor<T> =>
   (row: T) => {
     const key = column.key as keyof T;
-    return row[key];
+    const value = row[key];
+    return typeof value === 'function' ? value() : value;
```

Comment:
Will see later on that the values of a summary item are functions rather than concrete values so that they can't get out of sync